integration-cli/docker_cli_exec_test.go

```diff
@@ -618,8 +617,8 @@ func (s *DockerSuite) TestExecStartFails(c *check.C) {
 	name := "exec-15750"
 	dockerCmd(c, "run", "-d", "--name", name, "busybox", "top")
 	c.Assert(waitRun(name), check.IsNil)
-
 	out, _, err := dockerCmdWithError("exec", name, "no-such-cmd")
-	c.Assert(err, check.NotNil, check.Commentf(out))
-	c.Assert(out, checker.Contains, "executable file not found")
+	if !(err != nil || out == "" || strings.Contains(err.Error(), "Error running command in existing container")) {
+		c.Fatalf("exec with non-existent cmd should have errored out with no output")
```

Comment:
Doesn't this seem wrong to show no output at all? Not even an "exe not found" type of error msg.
Also, shouldn't we be checking the exit code to make sure its correct?